src/lib/reif.pl

```diff
@@ -98,3 +98,34 @@
 tmember_t(_P_2, [], false).
 tmember_t(P_2, [X|Xs], T) :-
    if_( call(P_2, X), T = true, tmember_t(P_2, Xs, T) ).
+
+%% if_(+If_1, +Then_0, +Else_0)
+%
+%  A monotonic if-then-else. When the last argument of the predicate If
+%  unifies with true, Then is executed, if the last argument
+%  unifies with false, Else is executed. 
+
+%% =(?X, ?Y, ?T)
+%
+%  Unifies T with false if X and Y are different terms. Otherwise, T is
```

Comment:
For example, this predicate reifies *syntactic&nbsp;equality* of terms: It turns the equality into a "thing", a Prolog term that we can reason about.